frontend/src/components/AverageTransactionFee/index.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect, useMemo, useState } from "react";
+import { TextLink } from "@stellar/design-system";
+import { useDispatch } from "react-redux";
+import { useRedux } from "hooks/useRedux";
+
+import "./styles.scss";
+
+import { SectionCard } from "components/SectionCard";
+import {
+  TimeRange,
+  VerticalBarChart,
+} from "components/charts/VerticalBarChart";
+import { LedgerTransactionHistoryFilterType } from "types";
+import { fetchAverageTransactionsFeeData } from "ducks/feeStats";
+
+const TIME_RANGE_MAPPING = [
+  {
+    key: TimeRange.DAY,
+    label: LedgerTransactionHistoryFilterType["24H"],
+  },
+  {
+    key: TimeRange.MONTH,
+    label: LedgerTransactionHistoryFilterType["30D"],
+  },
+];
+
+export const AverageTransactionFee: React.FC = () => {
+  const { feeStats } = useRedux("feeStats");
+  const dispatch = useDispatch();
+
+  const [rangeInterval, setRangeInterval] = useState(TimeRange.DAY);
+
+  useEffect(() => {
+    dispatch(fetchAverageTransactionsFeeData());
+  }, [dispatch]);
+
+  const data = useMemo(() => {
+    if (feeStats.fees) {
+      const formattedMonthlyFee = feeStats.fees.month.map((fee) => ({
+        date: new Date(fee.date),
```

Comment:
let's put the dates in `<day>/<month> <time>` format (so just switching the day and month order from the current format)

and for both the data in the chart and the x-axis format

![Screen Shot 2022-07-19 at 3 38 07 PM](https://user-images.githubusercontent.com/30449853/179860782-c33f5cc2-2ad7-4bc0-9429-523d7954ce34.png)

Reply:
Yes, we changed this format but in another PR, that's why it didn't reflect, when we merge we believe that all graphs will have the same date format, but I will change here too.

Reply:
Hey @acharb, we weren't clear on what this change meant, in this picture the format is already `DD/MM H:MM`, so we don't really know if it's a data issue on the backend or a display issue change.

Reply:
@acharb I didn't understand either, I think that x-axis dates are wrong so I fixed this, does make sense now?

<img width="847" alt="Screen Shot 2022-07-20 at 11 34 09" src="https://user-images.githubusercontent.com/54275445/180009037-421baaca-959d-4cdd-ba33-f7370a0c2383.png">

Reply:
oops sorry! I meant `<month>/<day>` format

like how the transactions per second chart is

<img width="602" alt="Screen Shot 2022-07-20 at 9 21 18 AM" src="https://user-images.githubusercontent.com/30449853/180033244-322db916-fa1d-4611-94dd-c8ef208ab512.png">